core-developers/experts.rst

```diff
@@ -209,7 +209,7 @@ sndhdr
 socket                gpshead
 socketserver
 spwd
-sqlite3               ghaering^
+sqlite3               ghaering^, erlend-aasland*
```

Comment:
Perhaps we should remove Gerhard. AFAIK, he isn't even maintaining the original pysqlite repo anymore.